practice-app/server/src/api/controllers/user/login.js

```diff
@@ -0,0 +1,36 @@
+import { User } from '../../../models/index.js';
+import { validateLogin } from '../../validators/user.validator.js';
+import bcrypt from 'bcryptjs';
+
+export default async (req,res) => {
+    const { error } = validateLogin(req.body);
+    if (error) 
+        return res.status(400).json({ "resultMessage": error.details[0].message });
+
+    const email = req.body.email;
+
+    let emailCheck = await User.exists({ email: email})
+        .catch((err) => {
+        return res.status(500).json({ "resultMessage": err.message });
+    });
+
+    if (!emailCheck) 
+        return res.status(404).json({"resultMessage": "There is no user with the given email."});
+
+    let user1 = await User.find({ email: email }).select('+password')
```

Comment:
You can directly use "findOne" method and so that you will not have to take the first element of an array in the next lines.

Reply:
I have changed that method as you suggested. Thank you for your review, @bahricanyesil !